src/components/loader-bar/loader-bar.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import LoaderBar from ".";
+
+export const DefaultStory: ComponentStory<typeof LoaderBar> = () => (
+  <LoaderBar mt={2} />
+);
+DefaultStory.storyName = "default";
```

Comment:
This is a minor thing, but as we specify the story name in the `stories.mdx` file, we don't have to do it again here, so we can remove the `.storyName` declarations in this file.